src/python/WMCore/Services/Requests.py

```diff
@@ -577,5 +577,5 @@ def __del__(self):
         try:
             # it'll likely fail, but give it a try
             shutil.rmtree(self.dir, ignore_errors=True)
-        except (IOError, OSError):
+        except:
```

Comment:
```except Exception``` splitting hairs, but this is allowed by our pylint setup, specifying nothing is not. We assume you are being especially lazy in the latter case.